src/main/java/org/jabref/gui/bibtexextractor/ExtractBibtexDialog.fxml

```diff
@@ -4,11 +4,18 @@
 <?import javafx.scene.control.DialogPane?>
 <?import javafx.scene.control.TextArea?>
 
+<?import javafx.scene.layout.VBox?>
+
 <DialogPane prefHeight="430.0" prefWidth="586.0" xmlns="http://javafx.com/javafx/8.0.171"
             xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.jabref.gui.bibtexextractor.ExtractBibtexDialog">
     <content>
-        <TextArea fx:id="input" minHeight="-Infinity" prefHeight="350.0" prefWidth="586.0"/>
+        <VBox fx:id="contentVbox" minHeight="-Infinity" prefHeight="200.0" prefWidth="100.0">
+        <children>
+            <TextArea fx:id="input" minHeight="-Infinity" prefHeight="350.0" prefWidth="586.0"/>
+        </children>
+        </VBox>
     </content>
-    <ButtonType fx:id="extractButtonType" buttonData="OK_DONE" text="%Extract"/>
+    <ButtonType fx:id="parseButtonType" buttonData="OK_DONE" text = "Add to current library"/>
```

Comment:
Personally, I think it suffices to only provide the option to add to the current database. I guess this is the usual use case, and in the worst case the user only needs to create a new library first. 

@JabRef/developers what do you think?